src/assets/BDDInGo.md

```diff
@@ -144,7 +144,7 @@ go 1.19
 and then run
 
 ```shell
-go get github.com/cucumber/godog/cmd/godog@v0.12.6
+go get github.com/cucumber/godog/godog@v0.12.6
```

Comment:
There is an extra `godog` near the end. It should be
`go get github.com/cucumber/godog@v0.12.6 `

Reply:
let's see if its correct now